LithoStrings.podspec

```diff
@@ -0,0 +1,140 @@
+#
+#  To learn more about Podspec attributes see https://guides.cocoapods.org/syntax/podspec.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |spec|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  These will help people to find your library, and whilst it
+  #  can feel like a chore to fill in it's definitely to your advantage. The
+  #  summary should be tweet-length, and the description more in depth.
+  #
+
+  spec.name         = "LithoStrings"
+  spec.version      = "0.0.1"
+  spec.summary      = "A Swift Library for working with Attributed Strings"
+
+  # This description is used to generate tags and improve search results.
+  #   * Think: What does it do? Why did you write it? What is the focus?
+  #   * Try to keep it short, snappy and to the point.
+  #   * Write the description between the DESC delimiters below.
+  #   * Finally, don't worry about the indent, CocoaPods strips it!
+  spec.description  = <<-DESC
+This library helps you return values from Attributed strings. It also helps you set attributed strings. It does a lot of heavy lifting.
+                   DESC
+
+  spec.homepage     = "https://github.com/cjc8/lithostrings"
+  # spec.screenshots  = "www.example.com/screenshots_1.gif", "www.example.com/screenshots_2.gif"
+
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See https://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  spec.license      = { :type => "MIT", :file => "LICENSE"}
+  # spec.license      = { :type => "MIT", :file => "FILE_LICENSE" }
+
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the authors of the library, with email addresses. Email addresses
+  #  of the authors are extracted from the SCM log. E.g. $ git log. CocoaPods also
+  #  accepts just a name if you'd rather not provide an email address.
+  #
+  #  Specify a social_media_url where others can refer to, for example a twitter
+  #  profile URL.
+  #
+
+  spec.author             = { "Remmington Damper" => "kincaiddamper@gmail.com" }
+  # Or just: spec.author    = "Remmington Damper"
+  # spec.authors            = { "Remmington Damper" => "kincaiddamper@gmail.com" }
+  # spec.social_media_url   = "https://twitter.com/Remmington Damper"
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  # spec.platform     = :ios
+  # spec.platform     = :ios, "5.0"
+
+    spec.ios.deployment_target = "15.0"
+    spec.swift_version = "5.5"
```

Comment:
And this to 5.1 I think (also in the project)

Reply:
@cjc8 

Will you explain why you don't want the deployment target and swift version to be the latest versions?

Reply:
They should be the lowest they can be while still being able to build. If someone tries to import this library but their app is for iPhone users with 13.0 for above (for example) then there app wouldn't build with this library.